packages/bpk-component-navigation-tab-group/src/BpkNavigationTabGroup.module.scss

```diff
@@ -116,3 +117,25 @@
     }
   }
 }
+
```

Comment:
We want all css inside the top-level classname, is that possible? (Following how other components do this)